### PR TITLE
MHP-2363 tools menu warning

### PR DESCRIPTION
**Pull request by mikeallisonJS**
~~pr to test with netlify, DO NOT MERGE~~

### CODE REVIEW

**Comment by jesusfreak3** [comment]
@OzzieOrca  ok fixed the modal issue, and a few others.  I also just quickly converted the vm to this.  let me know.

File: app/assets/javascripts/angular/components/organizationOverview/organizationOverview.component.js [hunk 1]
```diff
@@ -22,69 +22,82 @@ function organizationOverviewController(
...
-        if (!vm.loadDetails) {
+        if (!this.loadDetails) {
             // Abort before loading org details
             return;
         }
 
+        var vm = this;
         // Make groups and surveys mirror that property on the organization
-        $scope.$watch('$ctrl.org.groups', function() {
+        $scope.$watch('$ctrl.org.groups', function(a) {
```
Comment (OzzieOrca) [inline app/assets/javascripts/angular/components/organizationOverview/organizationOverview.component.js]: What's the `a` for? Can you make this an arrow function too? I think `this` will work fine inside after.

File: app/assets/javascripts/angular/components/organizationOverview/organizationOverview.component.js [hunk 2]
```diff
@@ -22,69 +22,82 @@ function organizationOverviewController(
...
 
         // The suborgs, people, and team are loaded by their respective tab components, not this component.
         // However, this component does need to know how many people and team members there are, so set the
         // people and team to a sparse array of the appropriate length.
         organizationOverviewService
-            .getSubOrgCount(vm.org)
+            .getSubOrgCount(this.org)
             .then(function(subOrgCount) {
-                vm.suborgs = new Array(subOrgCount);
+                this.suborgs = new Array(subOrgCount);
```
Comment (OzzieOrca) [inline app/assets/javascripts/angular/components/organizationOverview/organizationOverview.component.js]: I don't know that `this` will work right in a normal function. There are a couple other places below.

File: app/assets/javascripts/angular/components/organizationOverview/organizationOverview.component.js (hunk 1 above)
Comment (mikeallisonJS) [reply app/assets/javascripts/angular/components/organizationOverview/organizationOverview.component.js]: not sure how that even got there...

File: app/assets/javascripts/angular/components/organizationOverview/organizationOverview.component.js (hunk 2 above)
Comment (mikeallisonJS) [reply app/assets/javascripts/angular/components/organizationOverview/organizationOverview.component.js]: agreed, I was just trying to keep existing convention.

**Pull request merged**